JavaScript/sign-script.js

```diff
@@ -0,0 +1,77 @@
+// const emailInput = document.getElementsByClassName('email')[0];
+// const errorMessage = document.getElementsByClassName('error-message')[0];
+const emailInput = document.querySelectorAll('.email')[0];
+const passInput = document.querySelectorAll('.password')[0];
+const passConfirmInput = document.querySelectorAll('.password-confirm')[0];
+const errorMail = document.querySelectorAll('.error-mail')[0];
+const errorPass = document.querySelectorAll('.error-pass')[0];
+const errorPassConfirm = document.querySelectorAll('.error-passconfirm')[0];
+const eyeImagePass = document.querySelector('#eyeImagePass'); 
+const eyeImageConfirm = document.querySelector('#eyeImageConfirm'); 
+const pattern = /^[A-Za-z0-9_]+[A-Za-z0-9]*[@]{1}[A-Za-z0-9]+[A-Za-z0-9]*[.]{1}[A-Za-z]{1,3}$/;
+let link = '/folder';
+
+function checkEmail(emailType) {
+  if (emailInput.value === '') {
+    errorMail.textContent = '이메일을 입력해주세요.';
+    emailInput.classList.add('error'); // 에러 클래스 추가
+  } else if (!pattern.test(emailType)) {
+    errorMail.textContent = '올바른 이메일 주소가 아닙니다.';
+    emailInput.classList.add('error'); // 에러 클래스 추가
+  } else if (emailInput.value !== '') {
+    errorMail.textContent = '이메일을 확인해주세요.';
+    emailInput.classList.add('error'); // 에러 클래스 추가
+  } else if (emailInput.value === 'test@codeit.com' && passInput.value === 'codeit101') {
```

Comment:
[중요도: 높음] test@codeit.com, codeit101 이 2개는 테스트를 위한 상수값입니다. 이렇게 조건문에 직접 박아버리면 나중에 이 값을 변경하려할 때 찾기 조금 번거로울 수 있고 다른 사람들이 봤을 때 바로 이해못할 수도 있습니다.

최상단에 document.querySelectorAll 쓰셨던 위치까지 옮기셔서 선언 후 사용하시면 더 편하게 사용 가능합니다.
```suggestion
  (2~3번째 쯤 줄)
  const TEST_CORRECT_EMAIL = 'test@codeit.com';
  const TEST_CORRENT_PASSWORD = 'codeit101'


  } else if (emailInput.value === TEST_CORRECT_EMAIL  && passInput.value === TEST_CORRENT_PASSWORD ) {
```

Reply:
이해했습니다! 감사합니다.